crates/loro/src/lib.rs

```diff
@@ -302,6 +322,11 @@ impl LoroDoc {
         self.doc.commit_with(options)
     }
 
+    /// Set commit message for the current uncommitted changes
+    pub fn set_next_commit_message(&self, msg: &str) {
```

Comment:
What will happen if I `set next commit message` and then commit with another msg

Reply:
last write win